client/src/components/CurrentTeam.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import axios from 'axios';
+import ModuleBar from './ModuleBar.jsx';
+import TeamTotal from './TeamTotal.jsx';
+import TeamMember from './TeamMember.jsx'
+
+
+class CurrentTeam extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      people: this.props.test,
+      currentView: this.props.test.slice(0, 8),
```

Comment:
A little confused by the two sample data sets here. Will you clean up later, or use a guard before your component? or alternatively set a "fetched: true/false" parameter?

Reply:
Yes, will definitely remove sample data I was using to style my module! I was also testing my axios get so that's why there's 2 sample sets. Thanks for the review!